core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -151,9 +151,21 @@ END_FUNC wait_secondary
 LOCAL_FUNC reset , :
 UNWIND(	.fnstart)
 UNWIND(	.cantunwind)
+
+/*
+ * if built-in arguments are configured, no arguments are expected from
+ * loader; instead, required parameters are configured in build time.
+ */
+
+#if defined (CFG_TEE_BUILT_IN_ARGS)
+    ldr r4, =CFG_TEE_PAGEABLE_ADDR
```

Comment:
This should either be refused (probably via some `.mk` file) together with `CFG_WITH_ARM_TRUSTED_FW` or `generic_entry_a64.S` needs to be updated for consistency.